pkg/plugins/mercury/mercury_observation.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package mercury;
+option go_package = ".;mercury";
+
+// TODO: what about different report formats for different clients?
+message MercuryObservationProto {
+    uint32 timestamp = 1;
```

Comment:
did you consider using the  built in proto3 timestamp type? if so, why not choose it? 

using uint32 risks weird timezone issues. 

in case you haven't seen it, there is a nice golang package for converting back and forth btw time.Time and protobuf.Timestamp https://pkg.go.dev/google.golang.org/protobuf/types/known/timestamppb

Reply:
I did not consider it, mostly because I this is copied from the median plugin. This is always assumed to be a unix timestamp which is always UTC/GMT and does not have timezone issues.